packages/quantic/force-app/main/default/lwc/quanticResultQuickview/quanticResultQuickview.js

```diff
@@ -140,4 +164,27 @@ export default class QuanticResultQuickview extends LightningElement {
   get buttonLabel() {
     return this.hasNoPreview ? this.labels.noPreview : this.labels.openPreview;
   }
+
+  get buttonClass() {
+    let classNames = ['slds-button'];
+
+    if (!this.previewButtonVariant || !this.previewButtonLabel) {
+      if (!this.hasNoPreview) {
+        classNames.push('quickview__button-base');
+      }
+    }
+    else {
+      classNames.push(`slds-button_${this.previewButtonVariant}`);
+    }
+
+    return classNames.join(' ');
+  }
+
+  get buttonIconClass() {
+    return `slds-current-color ${!this.previewButtonLabel ? '' : 'slds-button__icon_right'}`;
+  }
```

Comment:
When it comes to handling class names, I often prefer to use an array instead of string interpolation.

```suggestion
  get buttonIconClass() {
    return [
      'slds-current-color',
      this.previewButtonLabel && 'slds-button__icon_right'
    ].join(' ');
  }
```